ui/styles/src/base/_form.scss

```diff
@@ -359,11 +359,6 @@ p.pods-add-file {
 		max-width: none;
 	}
 
-	/* WP fix */
```

Comment:
What did this fix and should we check all other interfaces for weirdness?

Reply:
No, removing this fix fixed the margins for checkboxes at smaller breakpoints. The original lines were very old. I think the WP core form styling has changed since then, so the "fix" was probably causing issues already some time ago.